src/settings.c

```diff
@@ -102,6 +104,19 @@ get_color_scheme (void)
   return g_variant_new_uint32 (color_scheme);
 }
 
+static GVariant *
+get_contrast_value ()
+{
+  SettingsBundle *bundle = g_hash_table_lookup (settings, "org.gnome.desktop.a11y.interface");
+  g_autofree char *theme = NULL;
```

Comment:
theme is unused

Reply:
and the compiler doesn't even throw a warning...

Reply:
It is the one downside to the `g_auto*` usage, its always used because its usually assigned to `NULL`.